src/clj/fluree/db/json_ld/transact.cljc

```diff
@@ -210,16 +210,11 @@
    (stage db nil txn parsed-opts))
   ([db fuel-tracker txn parsed-opts]
    (go-try
-     (let [ctx           (:context parsed-opts)
-           parsed-txn    (q-parse/parse-txn txn ctx)
-           db*           (if-let [policy-identity (perm/parse-policy-identity parsed-opts ctx)]
-                           (<? (perm/wrap-policy db policy-identity))
-                           db)
-           tx-state      (->tx-state db*)
-           flakes-ch     (generate-flakes db fuel-tracker parsed-txn tx-state)
-           fuel-error-ch (:error-ch fuel-tracker)
-           chans         (remove nil? [fuel-error-ch flakes-ch])
-           [flakes]      (alts! chans :priority true)]
-       (when (util/exception? flakes)
-         (throw flakes))
+     (let [ctx        (:context parsed-opts)
+           parsed-txn (q-parse/parse-txn txn ctx)
+           db*        (if-let [policy-identity (perm/parse-policy-identity parsed-opts ctx)]
+                        (<? (perm/wrap-policy db policy-identity))
+                        db)
+           tx-state   (->tx-state db*)
+           flakes     (<? (generate-flakes db fuel-tracker parsed-txn tx-state))]
```

Comment:
`generate-flakes` returns a tuple of `[db flakes]`, I think it would make sense to destructure it here rather than in the call to `flakes->final-db`, just so the name doesn't confuse anyone in the future. Or, rather than have to come up with a name for the new db (db-with-updated-namespace-codes? bleh), we could just call it a generic `result`.

Reply:
I left this as is because I'm planning on overhauling this whole namespace soon.